openbb_sdk/providers/utils/unit_test_generator.py

```diff
@@ -0,0 +1,163 @@
+"""The unit test generator for the fetchers."""
+import os
+from typing import Any, Dict, List
+
+from openbb_core.app.provider_interface import ProviderInterface
+from openbb_provider.abstract.fetcher import Fetcher
+from pydantic.fields import ModelField
+
+from openbb_sdk.providers.utils.credentials_schema import test_credentials
+
+
+def get_provider_fetchers(
```

Comment:
Could this function be simplified to something like this? Looks easier to build the fetcher dict without importing the module and then get the attribute!

```suggestion
def get_provider_fetchers():
    registry = RegistryLoader.from_extensions()
    provider_fetcher_map = Dict[str, Dict[str, Fetcher]] = {}

    for provider_name, provider_cls in registry.providers.items():
        provider_fetcher_map[provider_name] = {}
        for fetcher_name, fetcher_cls in provider_cls.fetcher_dict.items():
            provider_fetcher_map[provider_name][fetcher_name] = fetcher_cls

    return provider_fetcher_map
```